include/Base.h

```diff
@@ -0,0 +1,26 @@
+#ifndef BASE_H
+#define BASE_H
+#include <vector>
+
+
+class BaseOperations
+{
+	public:
+	int ID=100;
+	struct mystruct //!< Structure of all the student details
+	{
+		int id;
+		std::string name;
+    	std::string branch;
```

Comment:
correct indentation

Reply:
It is correclty showing in VScode, while pushing it to Git; it gets changed.

Reply:
check tab and single space